# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1345341791 14400
#      Sat Aug 18 22:03:11 2012 -0400
# Node ID efac5593b1de83a3339824f1881eccafdc87d9f4
# Parent  fa0cf7d870d759e0aa98e8d2c43791866a497d4a
* __pie__.m: Avoid syntax errors in matrix lists.

diff --git a/scripts/plot/private/__pie__.m b/scripts/plot/private/__pie__.m
--- a/scripts/plot/private/__pie__.m
+++ b/scripts/plot/private/__pie__.m
@@ -112,36 +112,36 @@ function hlist = __pie__ (caller, vararg
       set (h, "clim", [1, 2]);
     else
       set (h, "clim", [1, len]);
     endif
 
     if (strncmp (caller, "pie3", 4))
       ln = length (xn);
       zlvl = 0.35;
-      sx = repmat (xoff + [0, - sind(xn), 0], [2 1]);
-      sy = repmat (yoff + [0, cosd(xn), 0], [2 1]);
+      sx = repmat (xoff + [0, -sind(xn), 0], [2, 1]);
+      sy = repmat (yoff + [0, cosd(xn), 0], [2, 1]);
       sz = [zeros(1, ln + 2); zlvl * ones(1, ln + 2)];
       sc = i * ones (size (sz));
 
       hlist = [hlist;
-        patch (xoff + [0, - sind(xn)], yoff + [0, cosd(xn)], zeros (1, ln + 1), i);
-        surface (sx, sy, sz, sc);
-        patch (xoff + [0, - sind(xn)], yoff + [0, cosd(xn)], zlvl * ones (1, ln + 1), i);
-        text (xt, yt, zlvl, labels{i})];
+        patch(xoff + [0, -sind(xn)], yoff + [0, cosd(xn)], zeros (1, ln + 1), i);
+        surface(sx, sy, sz, sc);
+        patch(xoff + [0, -sind(xn)], yoff + [0, cosd(xn)], zlvl * ones (1, ln + 1), i);
+        text(xt, yt, zlvl, labels{i})];
 
     elseif (strncmp (caller, "pie", 3))
       if (xt > 0)
         align = "left";
       else
         align = "right";
       endif
 
-      hlist = [hlist; patch(xoff + [0, - sind(xn)], yoff + [0, cosd(xn)], i);
-               text (xt, yt, labels{i}, "horizontalalignment", align)];
+      hlist = [hlist; patch(xoff + [0, -sind(xn)], yoff + [0, cosd(xn)], i);
+               text(xt, yt, labels{i}, "horizontalalignment", align)];
 
     else
       error ("__pie__: unknown caller `%s'", caller);
     endif
   endfor
 
   addlistener (gca, "view", {@update_text_pos, hlist});
 
